### PR TITLE
Fixed styling for talks on mobiles

### PR DESCRIPTION
**Pull request by anuschka**
Fixed styling for talks on mobiles
- removed fixed width for name
- redefined margin for h2 in talkers

### CODE REVIEW

File: _sass/_talks.scss [hunk 1]
```diff
@@ -1,49 +1,56 @@
...
 
 .talk-role {
 	font: {
 		size: 0.7em;
-		weight: 400;
+		weight: 600;
 	};
 	color: $red-color;
-	position:relative;
+	color: $white-color;
```
Comment (jurecuhalev) [inline _sass/_talks.scss:24]: remove duplicate declarations

File: index.html [hunk 2]
```diff
@@ -75,23 +75,32 @@
 	{% assign posts = site.posts | sort:"name" %}
 	{% for post in posts %}
 		{% if post.role == 'talk' %}
-		<div class="talkers-one">
-			<a class="" href="{{ site.baseurl }}{{ post.url }}">
-				<div class="talk-info">
-						<h2 class="talker-name">
+		<div class="talkers-one" class="row">
+				<a class="col-xs-12 col-sm-6 col-md-4" href="{{ site.baseurl }}{{ post.url }}">
```
Comment (jurecuhalev) [inline index.html]: you don't need `col-xs-12` class, it's automatic if you have any other bootstrap class (like `col-sm-6`)

File: index.html [hunk 3]
```diff
@@ -75,23 +75,32 @@
...
+					<div class="talk-info">
+            <p class="talk-image">
+							<img src="{{ site.baseurl }}/assets/images/speakers/{{ post.avatar }}"></img>
```
Comment (jurecuhalev) [inline index.html]: `img` tags are self-closing, so: ``. There is no `</img>`

**Comment by jurecuhalev** [comment]
You have unclosed HTML tag somewhere: https://www.dropbox.com/s/xx2yy1pqjddv6q3/Screenshot%202016-02-08%2023.39.02.png?dl=0

**Comment by jurecuhalev** [comment]
Overall, I like it. I would put image and track name also on detail page.

File: _sass/_talkers.scss [hunk 4]
```diff
@@ -2,28 +2,27 @@
...
+		background-color: $grey-color;
 	}
 }
 
+
 .talker-name {
 	font: {
-		size: 0.9em;
-		weight: 400;
+		weight: 600;
```
Comment (jurecuhalev) [inline _sass/_talkers.scss]: This is strange. You probably meant `font-weight`, without the `font: {}`

File: _sass/_talks.scss [hunk 5]
```diff
@@ -1,49 +1,56 @@
...
 
 .talk-title {
 	font: {
-		size: 0.9em;
-		weight: 300;
-		family: $base-font-family;
-	};
-	padding: 0 0.25em;
-	color: $text-color;
+		weight: 400;
```
Comment (jurecuhalev) [inline _sass/_talks.scss]: see above

File: _sass/_talks.scss [hunk 6]
```diff
@@ -1,49 +1,56 @@
...
+		};
+	color: $red-color;
 	position:relative;
+	padding: 0 0 0 0.50em;
+	@media(max-width:767px) {
+      font-size: 0.8em;
+  }
+}
+.talk-bottom{
+	position: absolute;
```
Comment (jurecuhalev) [inline _sass/_talks.scss:16]: you have problems with your tabs/spaces and indents

File: _sass/_talks.scss [hunk 7]
```diff
@@ -1,49 +1,56 @@
...
+  bottom: 0;
+  left: 0;
 }
-
 
 .talk-role {
 	font: {
 		size: 0.7em;
-		weight: 400;
+		weight: 600;
```
Comment (jurecuhalev) [inline _sass/_talks.scss]: see above (font-weight)

Comment (jurecuhalev) [inline _sass/_talks.scss]: same for `font-size`

File: index.html (hunk 2 above)
Comment (anuschka) [inline index.html]: If I remove col-xs-12 the design breaks on phones.

File: index.html (hunk 3 above)
Comment (anuschka) [inline index.html]: I checked (several times index.html) and cannot find: 

> You have unclosed HTML tag somewhere: https://www.dropbox.com/s/xx2yy1pqjddv6q3/Screenshot%202016-02-08%2023.39.02.png?dl=0

**Pull request merged**